### PR TITLE
fix: Add error handling if default EA App path

### LINKED ISSUE #171: Detect default EA install location and give warning
"C:\Program Files\EA<something>" cannot be written to by other processes due to some Windows directory permission stuff.

To resolve, the solution is to move the game install to a different location.

We should detect such an issue and guide the user to the right location to resolve.

### PR DESCRIPTION
**Pull request by GeckoEidechse**
We cannot install to `C:\Program Files\EA Games\Titanfall2` cause Windows™
As such if install fails we check for that path and show according error message.

I would kinda also wanna add a link to wiki but AFAIK links won't render clickable so it's not really super helpful...

Closes #171

### CODE REVIEW

**Comment by AnActualEmerald** [comment]
A plain html link probably won't work but you can open links from within a tauri app https://tauri.app/v1/api/js/shell#open

**Comment by GeckoEidechse** [comment]
> > AFAIK links won't render clickable
> 
> WDYM? We have clickable links on Thunderstore mods cards, for instance.

Right now it's giving a system popup (which arguably is justified cause Northstar failing to install is kinda a big deal). But even if changed I'll still need to essentially stick the HTML into the returned string to show as a link.

At some point I need to refactor error returns to an enumerator so that we can localisation and all that but I first need to learn how to do that which will take me at least a week with everything else going on ^^"

Anyway, I'm happy to take any suggestions to improve the state in the short term :P

File: src-tauri/src/lib.rs [hunk 1]
```diff
@@ -239,12 +239,26 @@ pub async fn install_northstar(
...
         std::path::Path::new(game_path),
     )
     .await
-    .unwrap();
+    {
+        Ok(_) => (),
+        Err(err) => {
+            if game_path
+                .to_lowercase()
+                .contains(&r#"C:\Program Files\EA Games\Titanfall2"#.to_lowercase())
```
Comment (Alystrasz) [inline src-tauri/src/lib.rs:252]: Can the EA App games path be different from this one?

Comment (Alystrasz) [reply src-tauri/src/lib.rs:252]: Why not checking for `C:\Program Files\` only then?

Comment (GeckoEidechse) [reply src-tauri/src/lib.rs]: Fair enough, changed in 8992810e0fcce01ba9b7b0c480db8d80009e1f24

**Pull request merged**